x-pack/plugins/transform/public/app/sections/transform_management/components/transform_list/action_delete.tsx

```diff
@@ -71,17 +85,96 @@ export const DeleteAction: FC<DeleteActionProps> = ({ items, forceDisable }) =>
     defaultMessage: 'Delete {transformId}',
     values: { transformId: items[0] && items[0].config.id },
   });
-  const bulkDeleteModalMessage = i18n.translate(
-    'xpack.transform.transformList.bulkDeleteModalBody',
-    {
-      defaultMessage:
-        "Are you sure you want to delete {count, plural, one {this} other {these}} {count} {count, plural, one {transform} other {transforms}}? The transform's destination index and optional Kibana index pattern will not be deleted.",
-      values: { count: items.length },
-    }
+  const bulkDeleteModalMessage = (
+    <>
+      <p>
+        <FormattedMessage
+          id="xpack.transform.transformList.bulkDeleteModalBody"
+          defaultMessage="Are you sure you want to delete {count, plural, one {this} other {these}} {count} {count, plural, one {transform} other {transforms}}?"
+          values={{ count: items.length }}
+        />
+      </p>
+      <EuiFlexGroup direction="column" gutterSize="none">
+        <EuiFlexItem>
+          {
+            <EuiSwitch
+              data-test-subj="mlAnalyticsJobDeleteIndexSwitch"
+              style={{ paddingBottom: 10 }}
+              label={i18n.translate(
+                'xpack.transform.actionDeleteTransform.bulkDeleteDestinationIndexTitle',
+                {
+                  defaultMessage: 'Delete destination indexes',
+                }
+              )}
+              checked={deleteDestIndex}
+              onChange={toggleDeleteIndex}
+            />
+          }
+        </EuiFlexItem>
+        <EuiFlexItem>
+          {
+            <EuiSwitch
+              data-test-subj="mlAnalyticsJobDeleteIndexPatternSwitch"
+              label={i18n.translate(
+                'xpack.transform.actionDeleteTransform.bulkDeleteDestIndexPatternTitle',
+                {
+                  defaultMessage: 'Delete destination index patterns',
+                }
+              )}
+              checked={deleteIndexPattern}
+              onChange={toggleDeleteIndexPattern}
+            />
+          }
+        </EuiFlexItem>
+      </EuiFlexGroup>
+    </>
+  );
+
+  const deleteModalMessage = (
+    <React.Fragment>
```

Comment:
Nit: Can be just `<>`.

Reply:
Fixed in https://github.com/elastic/kibana/pull/67922/commits/d3c37b5c0d03c2ca6e211a3ff2380154223c2f0d